### PR TITLE
Add support for persitent volume caims

### PR DESCRIPTION
**Pull request by waok2**
This PR adds support for persistent volumes claims.

### CODE REVIEW

**Review by virtualroot — CHANGES_REQUESTED** [review]
Thanks @waok2 for your work. I added a few comments.

File: charts/structurizr/templates/pvc.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+spec:
+  storageClassName: {{ .Values.PersistentVolumeClaim.storageClass }}
+  accessModes:
+    {{- range .Values.PersistentVolumeClaim.accessModes }}
+    - {{ . }}
+    {{- end }}
+  resources:
+    requests:
+      storage: {{ .Values.PersistentVolumeClaim.size }}
+{{- end }}
```
Comment (virtualroot) [inline charts/structurizr/templates/pvc.yaml:15]:
```suggestion
{{- if .Values.persistence.enabled }}
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: {{ .Values.persistence.name }}
spec:
  storageClassName: {{ .Values.persistence.storageClass }}
  accessModes:
    {{- range .Values.persistence.accessModes }}
    - {{ . }}
    {{- end }}
  resources:
    requests:
      storage: {{ .Values.persistence.size }}
{{- end }}
```

File: charts/structurizr/values.yaml [hunk 2]
```diff
@@ -100,6 +100,17 @@ volumeMounts: []
   # - name: structurizr-bucket
   #   mountPath: /usr/local/structurizr
 
+PersistentVolumeClaim:
```
Comment (virtualroot) [inline charts/structurizr/values.yaml:103]:
```suggestion
persistence:
```

**Comment by waok2** [comment]
@virtualroot I have committed the changes you requested.

**Review by virtualroot — APPROVED** [review]

**Comment by waok2** [comment]
@virtualroot Do we have anything that is preventing you from merging the PR?

**Comment by virtualroot** [comment]
@waok2 merged, thanks!

**Pull request merged**